selfdrive/car/gm/values.py

```diff
@@ -113,6 +114,7 @@ def init_make(self, CP: car.CarParams):
     GMCarInfo("GMC Sierra 1500 2020-21", "Driver Alert Package II", video_link="https://youtu.be/5HbNoBLzRwE"),
   ],
   CAR.EQUINOX: GMCarInfo("Chevrolet Equinox 2019-22"),
+  CAR.TRAILBLAZER: GMCarInfo("Chevrolet Trailblazer 2020-22", "Adaptive Cruise Control"),
```

Comment:
there is no 2020 Trailblazer, the first model year was 2021. Do you mean the 2020 Blazer?

Reply:
no, I mean the Trailblazer (Crossover) model.

It was first sold in Korea on January 16, 2020, so I assume it is a 2020 model year.

Although there are references to it being sold in the U.S. market as a 2021 model year in early 2020, please correct me if there is any inaccuracy.

Reply:
Changed to 2021